applications/GeoMechanicsApplication/tests/test_dirichlet_u.py

```diff
@@ -0,0 +1,85 @@
+import os
+
+import KratosMultiphysics.KratosUnittest as KratosUnittest
+import test_helper
+
+class KratosGeoMechanicsDirichletUTests(KratosUnittest.TestCase):
+    """
+    This class contains some element tests, such as triaxial and oedometer tests
+    """
+
+    def setUp(self):
+        # Code here will be placed BEFORE every test in this TestCase.
+        pass
+
+    def tearDown(self):
+        # Code here will be placed AFTER every test in this TestCase.
+        pass
+
+    def test_dirichlet_u(self):
+        """
+        4 element elongation test in 2 stages. The incremental elastic material should show 
+        linear increase in stress_yy over the steps, with continuity over the stages.
+        To reach this a saw tooth like table for the prescribed (stage) displacement is given.
+        H = 1 [m]
+        Young's modulus E = 30E+06 [N/m^2]
+        stage 1) elongate 0.05 [m] in 2 steps
+        stage 2) elongate another 0.05 [0.05] in 2 steps
+        """
+        test_name    = 'test_dirichlet_u'
+        project_path = test_helper.get_file_path(os.path.join('.', test_name))
+        n_stages     = 2
+        stages       = test_helper.get_stages(project_path,n_stages)
+
+        # name of output file
+        output_file_names = [os.path.join(project_path, 'test_dirichlet_u_stage' + str(i + 1) + '.post.res') for i in
+                            range(n_stages)]
+        output_data       = []
+
+        # run stages and get results
+        for idx, stage in enumerate(stages):
+            stage.Run()
+            reader = test_helper.GiDOutputFileReader()
+            output_data.append(reader.read_output_from(output_file_names[idx]))
```

Comment:
We can use zip here, to iterate over the two lists at the same time (a trick I learned from @avdg81), i.e.:
```suggestion
        for output_file_name, stage in zip(output_file_names, stages):
            stage.Run()
            reader = test_helper.GiDOutputFileReader()
            output_data.append(reader.read_output_from(output_file_name))
```

(But please check if I didn't make any mistakes/typos in this suggestion 😄 )

Reply:
done